patch/files/scripts/outside.gd

```diff
@@ -563,31 +564,44 @@ func outskirts():
 ############## person GUILD
 
 
-func newslaveinguild(number, town = 'wimborn'):
+func newslaveinguild(number, town = 'wimborn', raceadd = 'Human'): #ralph5 added 3rd entry: raceadd

```

Comment:
The arg "raceadd" should have a default value of 'rand' as that is consistent with prior usage. Changes are made to the racial distribution without condition. The random selection of races and origins has regressed. Specified values of 'raceadd' should override slaverguildallraces. originpool is uneccesarily duplicated based on 'raceadd' which convolutes its purpose.

Reply:
Ill come back to this

Reply:
I changed the structure of this in the way I thought was best, please look at this again when you get the chance.

Reply:
I broke it lol

Reply:
I'm not sure what the purpose of rand is at this moment. Though I think its supposed to expand the functionality of newslaveinguild so you can call it while defining a specific race. I'll upload a change and see what you think.